docs/source_values.rst

```diff
@@ -65,13 +70,16 @@ device will iterate over the values of the device provided, setting the
 device's value to each element at a rate specified in the
```

Comment:
I wonder if the paragraph above should have links (perhaps to the python docs) explaining the terms "generator" and "iterator" ?

Reply:
Done - in next push